lint-manifest.yml

```diff
@@ -0,0 +1,17 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: 18fgsa/concourse-task
+
+inputs:
+- name: pipeline-config
```

Comment:
I think these will still always require `input_mapping` in the pipeline if setup as a new generic task in each cg-deploy-* (https://github.com/18F/cg-deploy-nessus-manager/pull/17) so not sure what that would gain us. If we don't do in pipeline tasks adding to the end of each manifest generation script sounds like it will be cleanest.